docs/source/WritingTests.rst

```diff
@@ -960,6 +960,10 @@ Avocado offers some options for the test writers to skip a test:
 Test ``skip()`` Method
 ----------------------
 
+.. warning:: `self.skip()` will be deprecated by the end of 2017.
```

Comment:
I meant close to the end of the year. Like a Christmas gift :)
But I can change it. What would you prefer?